python/semantic_kernel/services/kernel_services_extension.py

```diff
@@ -68,7 +68,7 @@ def get_service(
         self,
         service_id: str | None = None,
         type: type[AI_SERVICE_CLIENT_TYPE] | tuple[type[AI_SERVICE_CLIENT_TYPE], ...] | None = None,
-    ) -> AIServiceClientBase:
```

Comment:
Why does this need to change?

Reply:
it fixes type inferencing, made me wonder why we hadn't done that before...